apps/web/src/components/views/ProfileView/components/ProfileLanding.tsx

```diff
@@ -1,59 +1,72 @@
+import { Icon } from "@dotkomonline/ui"
 import StudentProgress from "@/components/molecules/StudentProgress/StudentProgress"
 import { Avatar } from "@radix-ui/react-avatar"
 import { NextPage } from "next"
 import { User } from "next-auth"
 
 interface IFormInput {
   name: string
-  value?: string | null | undefined
+  children?: JSX.Element
+  addMore?: string
+  clickable?: boolean
 }
 
-const FormInput: React.FC<IFormInput> = ({ name, value }) => {
+const FormInput: React.FC<IFormInput> = ({ name, children, addMore, clickable = true }) => {
   return (
-    <>
-      <div className="flex w-[450px] items-center justify-between">
+    <div className="my-10 ">
+      <div className="ml-4">
         <label>{name}</label>
-        <p className="text-slate-12 flex min-h-[37px] items-center">{value ?? name}</p>
       </div>
-      <hr className="border-slate-12 my-5 w-full" />
-    </>
+      <hr className="border-slate-12 w-full opacity-50" />
+      <div className="ml-10 w-2/3 space-y-3 ">
+        <div
+          className={`mt-3 flex items-center justify-between rounded-lg pl-2 ${
+            clickable ? "hover:bg-slate-3 hover:cursor-pointer" : ""
+          }`}
```

Comment:
If you find it annoying to write concatenated class names, you can try the cn function https://github.com/dotkom/monoweb/blob/main/packages/ui/src/utils.ts#L4 to concatenate class names conditionally
```suggestion
          className={cn('mt-3 flex items-center justify-between rounded-lg pl-2', clickable && 'hover:bg-slate-3 hover:cursor-pointer')}
```